cmd/harvest/version/version.go

```diff
@@ -25,3 +37,89 @@ func String() string {
 		runtime.GOARCH,
 	)
 }
+
+func Cmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "version",
+		Short: "Show application version and check for latest",
+		Long:  "Show application version and check for latest",
+		Run:   doVersion,
+	}
+}
+
+func doVersion(cmd *cobra.Command, _ []string) {
+	fmt.Printf(cmd.Root().Version)
+	checkLatest()
+}
+
+func checkLatest() {
+	//goland:noinspection GoBoolExpressions
+	if Commit == "HEAD" {
+		return
+	}
+	fmt.Printf("checking GitHub for latest...")
+	latest, err := latestRelease()
+	if err != nil {
+		fmt.Printf("%s\n", err)
+		return
+	}
+	available, err := isNewerAvailable(Release, latest)
+	if err != nil {
+		fmt.Printf("%s\n", err)
+		return
+	}
+	if available {
+		fmt.Printf(" newer version %s available at %s\n", latest, githubReleases)
+	} else {
+		fmt.Printf(" %s\n", youHaveLatest)
+	}
+}
+
+func isNewerAvailable(current string, remote string) (bool, error) {
+	//fmt.Printf("isNewerAvail cur=%s remote=%s ", current, remote)
+	if remote == current {
```

Comment:
i think remote != "" && remote != current would also work. No need to check greaterorequal

Reply:
that might work - the local build complicated the logic